util/src/main/i18n/templates/match.properties

```diff
@@ -52,19 +52,56 @@ match.stats = Kills: {0}  |  Killstreak: {1}  |  Deaths: {2}  |  K/D: {3}
 # {1} = an amount of kills(number)
 match.stats.kills = Kills: {1} by {0}
 
+# {0} = an amount of kills(number)
+match.stats.kills.concise = Kills: {0}
+
 # {0} = a player name
 # {1} = a killstreak(number)
 match.stats.killstreak = Killstreak: {1} by {0}
 
+# {0} = a killstreak(number)
+match.stats.killstreak.concise = Highest killstreak: {0}
+
 # {0} = a player name
 # {1} = an amount of deaths(number)
 match.stats.deaths = Deaths: {1} by {0}
 
+# {0} = an amount of deaths(number)
+match.stats.deaths.concise = Deaths: {0}
+
+# {0} = numbers of kills divided by the number of deaths (kill-death ratio)
+match.stats.kd.concise = K/D: {0}
+
 # {0} = a player name
 # {1} = an amount of blocks
 match.stats.bowshot = Longest Shot: {1} blocks by {0}
 
-match.stats.overall = Match Stats
+# {0} = a player name
+# {1} = an amount of damage
+match.stats.damage = Damage: {1} by {0}
```

Comment:
Melee Damage

Reply:
Actually, not sure what this is for?

Reply:
This would appear on the "Best of" stats at the end of the match.
It takes all damage done by that player towards other players.